### ISSUE TITLE
Document how to backup and restore your application directory

### ISSUE THREAD

**Issue by woodser** [haveno-dex/haveno#1279]
This issue requests documenting how to backup and restore your application directory in the [docs](https://github.com/haveno-dex/haveno-docs) repository, as it's a frequently asked question.

To summarize:

Go to Account > Backup > Select backup location.

This will save your entire application directory to a location of your choice.

The backup includes your main wallet and the current state of your offers or trades. You should create a new backup in order to include any new or updated offers or trades.

To restore, you can install Haveno, then copy your backup folder as your main application folder, located at:

- Linux: ~/.local/share/Haveno/
- macOS: ~/Library/Application Support/Haveno/
- Windows: ~\AppData\Roaming\Haveno\

Note: replace "Haveno" with the name of the third party network you're using, if applicable.

**Comment by woodser** [comment]
Related issue: https://github.com/haveno-dex/haveno-docs/issues/16

**Comment by Newaccount-github** [comment]
> This issue requests documenting how to backup and restore your application directory in the [docs](https://github.com/haveno-dex/haveno-docs) repository, as it's a frequently asked question.
> 
> To summarize:
> 
> Go to Account > Backup > Select backup location.
> 
> This will save your entire application directory to a location of your choice.
> 
> The backup includes your main wallet and the current state of your offers or trades. You should create a new backup in order to include any new or updated offers or trades.
> 
> To restore, you can install Haveno, then copy your backup folder as your main application folder, located at:
> 
> * Linux: ~/.local/share/Haveno/
> * macOS: ~/Library/Application Support/Haveno/
> * Windows: ~\AppData\Roaming\Haveno\
> 
> Note: replace "Haveno" with the name of the third party network you're using, if applicable.



This helps a lot. Can using the share app on windows and then emailing it to yourself also save your data? Or does that only share the app itself? 

Thanks

**Comment by woodser** [comment]
> Can using the share app on windows and then emailing it to yourself also save your data? Or does that only share the app itself?

You can backup your application directory any way you like, though it's probably too big to be sent over email, since it can be 100s of mbs.

Be sure to backup your application directory and not the application itself, by creating a backup from within the app or by copying your application directory manually to a location of your choice. The application directory is located at:

- Linux: ~/.local/share/Haveno/
- macOS: ~/Library/Application Support/Haveno/
- Windows: ~\AppData\Roaming\Haveno\

**Comment by Newaccount-github** [comment]
> > Can using the share app on windows and then emailing it to yourself also save your data? Or does that only share the app itself?
> 
> You can backup your application directory any way you like, though it's probably too big to be sent over email, since it can be 100s of mbs.
> 
> Be sure to backup your application directory and not the application itself, by creating a backup from within the app or by copying your application directory manually to a location of your choice. The application directory is located at:
> 
> * Linux: ~/.local/share/Haveno/
> * macOS: ~/Library/Application Support/Haveno/
> * Windows: ~\AppData\Roaming\Haveno\

This helps a lot. One more question: if the Monero is in the security deposit, your seed phrase and date would not be able to recover it correct?

**Comment by woodser** [comment]
Correct, because the funds are held in a multisig wallet, so your backup needs to contain that wallet for you to recover the funds by completing the trade.

**Comment by jermanuts** [comment]
https://docs.haveno.exchange/haveno-ui/backup_and_restore
